src/Chorus/UI/Settings/SendReceiveSettings.resx

```diff
@@ -128,8 +128,8 @@
     <value>&lt;!DOCTYPE html&gt;&lt;html&gt;&lt;body style='font-family:"Microsoft Sans Serif","sans-serif";font-size:9pt'&gt;
 &lt;p style='margin:0in;'&gt;Chorus Hub is a simple way to Send/Receive on a local network, like in an office or home. These instructions are basic, but they should be enough to get you started.&lt;/p&gt;
 &lt;ol&gt;
-&lt;li style='margin-left:-10pt'&gt;Designate one computer on your network as "the server", and use the following instructions to make sure this product is installed and running on it.&lt;/li&gt;
-&lt;li style='margin-left:-10pt'&gt;
+&lt;li style='margin-left:-10pt'&gt;Designate one computer on your network as "the server".&lt;/li&gt;
+&lt;li style='margin-left:-10pt'&gt;Install and run Chorus Hub on "the server".
```

Comment:
Just installing the CH service on Windows, runs it. There is no more action needed by the user, other than to keep the 'server' running, when CH is used for S/R.